### DISCUSSION TITLE
[Feature Request] Dropped item (stacking) holograms?

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#7257] [Feature Requests]
Would it be possible to have a per player option to show items that are on the floor to show a hologram if you get nearby them? Optionally, try to stack dropped items. I am not looking for a server performance increase (but certainly not a decrease) with this. It's a visual helper for players for me personally at most. 

[ 17x cobblestone ] 
[ 1x diamond pickaxe ] 
[ 3x chicken spawn egg ]

and the holograms disappear again as I walked further away from them. if they are behind a wall, they don't need to show.

**Comment by MrCrazys** [comment]
I feel like this is a bit out of the scope of CMI. There are plenty of plugins out there that do and it already. I personally use [UltimateStacker](https://marketplace.songoda.com/marketplace/product/ultimatestacker-the-simple-to-use-stacker.16), which has support for CMI holograms. Regardless, this is just my opinion.
